modules/integration/tests-ui-integration/pom.xml

```diff
@@ -277,6 +277,11 @@ under the License.
             <artifactId>org.wso2.carbon.integration.common.utils</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.wso2.carbon.automationutils</groupId>
+            <artifactId>org.wso2.carbon.integration.common.admin.client</artifactId>
+            <version>4.3.0</version>
```

Comment:
Don't hard code the versions - User properties appropriately to hold version values. Version numbers should be defined in root/parent pom.